boot/freeldr/freeldr/arch/i386/i386bug.c

```diff
@@ -115,9 +116,11 @@ i386PrintExceptionText(ULONG TrapIndex, PKTRAP_FRAME TrapFrame, PKSPECIAL_REGIST
     i386_ScreenPosX = 0;
     i386_ScreenPosY = 0;
 
-    PrintText("An error occured in " VERSION "\n"
+    PrintText("Error in %s %s\n"
               "Report this error on the ReactOS Bug Tracker: https://jira.reactos.org\n\n"
-              "0x%02lx: %s\n\n", TrapIndex, i386ExceptionDescriptionText[TrapIndex]);
+              "0x%02lx: %s\n\n",
+              KERNEL_VERSION_STR, KERNEL_VERSION_BUILD_STR,
```

Comment:
```suggestion
```

Reply:
Resolved. I applied your suggestion with stripping the %s %s. It still works.
![image](https://github.com/user-attachments/assets/17e3fa9f-9c99-456b-ba19-6da317e1a1fc)